### ISSUE TITLE
Unable to progress past first Gate after Character Creation

### ISSUE THREAD

**Issue by drew-id** [DreamsideInteractive/FrozenFlameServer#14]
Replicated this in Local game and immediately after the gate cinematic, progresses through to the first teleport pad.

In the Server (Windows Server - if it matters), after making the character and the cinematic plays - you are left standing in front of the gate with it still closed.

Tried the following:
- creating save files in Local game and copying these to the server. Server started but could not join, kept looping back to server screen in menu

Any ideas?

**Comment by StevenZeng96** [comment]
Yes, knowing which system you're using does matter here. The simplest way is to [install Steam App on your server](https://store.steampowered.com/about) which could let the `FrozenFlameServer.exe` could get the user ID from Steam API.

Anyway, it's one of the issues we find during our test inside the Chinese publishing team, and I believe this is already listed in FAQ.

**Comment by drew-id** [comment]
> Yes, knowing which system you're using does matter here. The simplest way is to [install Steam App on your server](https://store.steampowered.com/about) which could let the `FrozenFlameServer.exe` could get the user ID from Steam API.
> 
> Anyway, it's one of the issues we find during our test inside the Chinese publishing team, and I believe this is already listed in FAQ.

I've installed the steam app on the server, does it need to be logged into because that's not going to work long term
Just installing it and running the server has made no change - i've attached logs below that occur directly after the cinematic plays at the door.

`[2022.11.25-09.54.11:437][461]LogNet: NotifyAcceptingChannel Control 0 server World /Game/FrozenFlame/Maps/Heaven/Heaven.Heaven: Accepted
[2022.11.25-09.54.11:439][461]LogNet: Remote platform little endian=1
[2022.11.25-09.54.11:439][461]LogNet: This platform little endian=1
[2022.11.25-09.54.11:637][462]LogNet: Login request: ?SkipTutorial?SteamAuth=1400000075B9CB4673E3E2956FCE6402010010019CC87E631800000001000000020000007BE9A6781D36C3FC78EF0C0D28000000B800000038000000040000006FCE64020100100188EA0A00C2FA46734501010A0000000038AA7D63B859996301001F290300010020941400000000007AD72BA460892A4B9DAA7FCE398B3B73E9185AB367875053F929B100802CC82BA82F3F44E052BF77587F4BC03D9CEE95DA02AC9273E34B529CF168416FD9F3C2FD4FFD03155C7508073F06E60BD3708A56AF49EFC3357CB6AEBE491CEA116B175EB812B64161A8165A6953B86F8C847992AE37249271A2D9A6A4322C45BFCEDF?Name=Talonical userId: Steam:76561198000426607 platform: Steam
[2022.11.25-09.54.11:640][462]LogAdvancedAuth: Warning: SteamGameServer() is nullptr
[2022.11.25-09.54.11:642][462]LogNet: PreLogin failure: ADVANCED_AUTHENTICATION_FAILURE
[2022.11.25-09.54.11:837][463]LogNet: UChannel::ReceivedSequencedBunch: Bunch.bClose == true. ChIndex == 0. Calling ConditionalCleanUp.
[2022.11.25-09.54.11:839][463]LogNet: UChannel::CleanUp: ChIndex == 0. Closing connection. [UChannel] ChIndex: 0, Closing: 0 [UNetConnection] RemoteAddr: 10.1.1.69:51477, Name: IpConnection_2147465967, Driver: GameNetDriver IpNetDriver_2147482449, IsServer: YES, PC: NULL, Owner: NULL, UniqueId: Steam:76561198000426607
[2022.11.25-09.54.11:841][463]LogNet: UNetConnection::Close: [UNetConnection] RemoteAddr: 10.1.1.69:51477, Name: IpConnection_2147465967, Driver: GameNetDriver IpNetDriver_2147482449, IsServer: YES, PC: NULL, Owner: NULL, UniqueId: Steam:76561198000426607, Channels: 2, Time: 2022.11.24-22.54.11
[2022.11.25-09.54.11:841][463]LogNet: UChannel::Close: Sending CloseBunch. ChIndex == 0. Name: [UChannel] ChIndex: 0, Closing: 0 [UNetConnection] RemoteAddr: 10.1.1.69:51477, Name: IpConnection_2147465967, Driver: GameNetDriver IpNetDriver_2147482449, IsServer: YES, PC: NULL, Owner: NULL, UniqueId: Steam:76561198000426607
[2022.11.25-09.54.12:037][464]LogNet: UNetConnection::PendingConnectionLost. [UNetConnection] RemoteAddr: 10.1.1.69:51477, Name: IpConnection_2147465967, Driver: GameNetDriver IpNetDriver_2147482449, IsServer: YES, PC: NULL, Owner: NULL, UniqueId: Steam:76561198000426607 bPendingDestroy=0 
[2022.11.25-09.54.34:513][576]LogTemp: WorldSaved, 52 Entities, 0 ModuleClusters`

**Comment by StevenZeng96** [comment]
> > Yes, knowing which system you're using does matter here. The simplest way is to [install Steam App on your server](https://store.steampowered.com/about) which could let the `FrozenFlameServer.exe` could get the user ID from Steam API.
> > Anyway, it's one of the issues we find during our test inside the Chinese publishing team, and I believe this is already listed in FAQ.
> 
> I've installed the steam app on the server, does it need to be logged in because that's not going to work long term Just installing it and running the server has made no change - I've attached logs below that occur directly after the cinematic plays at the door.

Nope, installing the Steam App on the server would be enough, no need to log in.
From what the report says, it only indicated they successfully check your id from SteamAPI, but fail to get authentication. My suggestion would be to try reinstalling the SeverTool from SteamCMD, and try reinstalling the game on your PC from Steam might be helpful.

**Comment by JEREMIE-HUMBERT** [comment]
Hello, 

I have the same issue.
do you have any idea to fix this problem? please

I already tried everything. 

Thank you for help and support.

**Comment by Freki-Geri** [comment]
is there a way to just disable steam networking? I don't really need it since I have the server prvt and only my friends can play
